frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSettings.tsx

```diff
@@ -17,19 +17,22 @@ export function EmbeddingSettings({
   updateSetting,
 }: AdminSettingComponentProps) {
   function handleToggleStaticEmbedding(event: ChangeEvent<HTMLInputElement>) {
```

Comment:
> https://v6.mantine.dev/core/switch/#controlled - we should be using `currentTarget`

these function calls are coming from Mantine switches so we should continue to use `currentTarget`